benchmarks/benchmarks/stats.py

```diff
@@ -57,44 +57,52 @@ def time_ttest_ind_diff_var(self):
 
 
 class Distribution(Benchmark):
+    # add distributions here
+    dists = ['cauchy', 'gamma', 'beta']
+
     param_names = ['distribution', 'properties']
     params = [
-        ['cauchy', 'gamma', 'beta'],
-        ['pdf', 'cdf', 'rvs', 'fit']
+        dists, ['pdf', 'logpdf', 'cdf', 'logcdf', 'rvs', 'fit', 'sf', 'logsf',
+                'ppf', 'isf', 'moment', 'stats', 'entropy', 'median', 'mean',
+                'var', 'std']
     ]
+    distcont = dict(distcont)
+    # maintain previous benchmarks' values
+    custom_input = {'gamma': [5], 'beta': [5, 3]}
 
     def setup(self, distribution, properties):
-        np.random.seed(12345678)
-        self.x = np.random.rand(100)
+        self.method = getattr(getattr(stats, distribution), properties)
+        x = np.random.rand(100)
+        shapes = self.distcont[distribution]
+        args = [x, *self.custom_input.get(distribution, shapes)]
+        names = ['loc', 'scale']
+        values = [4, 10]
+        kwds = dict(zip(names, values))
+
+        if properties == 'fit':
+            # provide only the data to fit in args
+            self.args = args[:1]
+        elif properties == 'rvs':
+            # add size for creation of data
+            kwds['size'] = 1000
+            self.args = args[1:]
+        elif properties == 'ppf' or properties == 'isf':
+            # picking arbitrary value for this
+            self.args = [.99, *args[1:]]
+        elif properties == 'moment':
+            self.args = [10, *args[1:]]
+        elif properties == 'stats':
+            self.args = args[1:]
+            kwds['moments'] = 'mv'
```

Comment:
Could you split this up into five tests? One for each of `m`, `v`, `s`, and `k`, and finally one for `mvsk`. I think this will help us look for issues like gh-11742.